src/ImageSharp/Colors/Spaces/YCbCr.cs

```diff
@@ -117,13 +121,7 @@ public YCbCr(byte y, byte cb, byte cr)
         /// <inheritdoc/>
         public override int GetHashCode()
         {
-            unchecked
-            {
-                int hashCode = this.Y.GetHashCode();
-                hashCode = (hashCode * 397) ^ this.Cb.GetHashCode();
-                hashCode = (hashCode * 397) ^ this.Cr.GetHashCode();
-                return hashCode;
-            }
+            return GetHashCode(this);
```

Comment:
can't this just be `return this.backingVector.GetHashCode()` directly and drop the `GetHashCode(...)` method? I don't think the indirection is needed.